src/io/flutter/preview/PreviewView.java

```diff
@@ -616,14 +628,22 @@ private void scrollTreeToNodeOnTop(TreeNode node) {
     }
   }
 
+  private void sendAnalyticEvent(@NotNull String name) {
+    FlutterInitializer.getAnalytics().sendEvent("flutterPreview", name);
```

Comment:
`flutterPreview` ==> `preview`?

Reply:
I was not sure if we get enough information to disambiguate it from other possible previews.
Fixed.